src/DIRAC/WorkloadManagementSystem/DB/JobDBUtils.py

```diff
@@ -112,7 +112,7 @@ def checkAndPrepareJob(jobID, classAdJob, classAdReq, owner, ownerGroup, jobAttr
         if result["Value"]:
             classAdReq.insertAttributeVectorString("Platforms", result["Value"])
         else:
-            error = "OS compatibility info not found"
+            error = "OS compatibility info not foud"
```

Comment:
?

Reply:
It was a debugging change to try and figure out where an error was coming from.